articles/quickstart/spa/_includes/_calling_api_create_backend_no_proxy.md

```diff
@@ -0,0 +1,65 @@
+<!-- markdownlint-disable MD002 MD041 -->
```

Comment:
This is a copy of `_calling_api_create_backend.md` with the following changes: 374b675c6218385e0fa86bf1cdbd34b714f3a2d2